sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -626,6 +624,74 @@ case class DescribeTableCommand(
   }
 }
 
+/**
+ * A command to list the info for a column, including name, data type, column stats and comment.
+ * This function creates a [[DescribeColumnCommand]] logical plan.
```

Comment:
This comment line seems not needed.

Reply:
There are other two similar comments (`ShowPartitionsCommand`, `ShowColumnsCommand`) in this file, shall I remove them all?

Reply:
A followup PR to improve the comments is sent: https://github.com/apache/spark/pull/19213